src/servicio.ts

```diff
@@ -0,0 +1,22 @@
+import Duration from "https://deno.land/x/durationjs@v4.0.0/mod.ts"; 
+
+
+export class Servicio{
```

Comment:
```suggestion
export class TiempoServicio{
```
Para solucionar el conflicto con `enum Servicio` de #11